test/layout/test_standard_pickup_state.py

```diff
@@ -102,3 +114,120 @@ def test_blank_as_json():
 
 def test_blank_from_json():
     assert StandardPickupState.from_json({}) == StandardPickupState()
+
+
+def test_check_consistency(mocker: pytest_mock.MockerFixture):
+    pickup = MagicMock()
+    pickup.game = RandovaniaGame.METROID_PRIME_ECHOES
+
+    # Check Amount of items shuffled
+    pickup.name = "Progressive Suit"
+    pickup.starting_condition = StartingPickupBehavior.CAN_BE_STARTING
+    state = StandardPickupState(num_shuffled_pickups=100)
+    expected_error = f"Can only shuffle between 0 and {DEFAULT_MAXIMUM_SHUFFLED[-1]} copies,"
+    f" got {state.num_shuffled_pickups}. ({pickup.name})"
+    escaped_error = re.escape(expected_error)
+
+    with pytest.raises(ValueError, match=escaped_error):
+        StandardPickupState.check_consistency(state, pickup)
+
+    state = StandardPickupState(num_shuffled_pickups=-1)
+    with pytest.raises(ValueError, match=escaped_error):
+        StandardPickupState.check_consistency(state, pickup)
+
+    # Check starting condition
```

Comment:
ok yeah, each time you have a comment like this dividing this test into chunks, you should just make a new dedicated test function for it